portfolio/src/main/webapp/index.html

```diff
@@ -1,56 +1,54 @@
 <!DOCTYPE html>
-<html>
+<html lang="en">
   <head>
-    <meta charset="UTF-8">
+    s<meta charset="UTF-8">
     <title>My Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
     <div id="content">
-        <div id = "nav_bar">
+        <div id = "nav-bar">
             <button >Home</button>
-            <div id="scroll_buttons">
-            <button>About</button>
-
-            <button>Experience</button>
-            <button>Education</button>
-            <button>Projects</button>
-            <button>Interests</button>
+            <div id="scroll-buttons">
+            <button id="about-button" onclick="moveToAbout()">About</button>
+            <button id="experience-button" onclick="moveToExperience()">Experience</button>
+            <button id="education-button" onclick="moveToEducation()">Education</button>
+            <button >Projects</button>
+            <button id="interests-button" onclick="moveToInterest()">Interests</button>
             </div>
         </div>
-        <div class="flex_hor">
+        <div class="flex-hor">
             <div>
                 <h1 style="margin-top:50%;"> HI ! I am </h1>
-                <h1 style="color:#fca61f";>Nourish Cherish</h1>
-                <h3>I dont just code. I also teach computers how to compile code</h2>
+                <h1 style="color:#fca61f;">Nourish Cherish</h1>
```

Comment:
[3.1.5](https://google.github.io/styleguide/htmlcssguide.html#Separation_of_Concerns): Separate structure from presentation from behavior.

Here and elsewhere (look for `style=` properties).

I see this color appears in multiple places. Instead of hardcoding it multiple times (which makes it difficult to change later because you need to remember to update each spot where you hardcoded it) I suggest making a CSS class that has this instead and setting that `class=` for all the elements you want to have the same color.